### ISSUE TITLE
Inform people about translating in the 'Language' option

### ISSUE THREAD

**Issue by Nubbie** [OpenRCT2/OpenRCT2#3736]
**Why:** 
- Get more translators
- Inform people about the opportunity
- Updates to the 'out of date' existing files like [ru-RU.txt](https://github.com/OpenRCT2/Localisation/blob/master/data/language/ru-RU.txt)

**Concept/Example:**
![translaters](https://cloud.githubusercontent.com/assets/17282384/15556060/22ab55d8-22cc-11e6-86d2-7a3a76abe7c6.png)

**Text used in the picture:**

```
STR_5830    :{SMALLFONT}{BLACK}Changes what language is used{Newline}Note: not all languages are complete{NEWLINE}{NEWLINE}You can help out with translations at{NEWLINE}Github.com/OpenRCT2/Localisation
```

**_Note: Text in the 'bottom' off the window really sticks out more than a tooltip_**

**Comment by marijnvdwerf** [comment]
Isn't this misuse of a tooltip?

**Comment by Nubbie** [comment]
Think there are some other strings used for information purpose or some sort of text that ain't a tip, wish I could skem through the language right now but it's too late, will edit this comment tomorrow

Still for 'a good cause' :p
Think a 'bottom note' might be better

**Comment by YJSoft** [comment]
Bottom link+"Note: not all languages are complete" tooltip would better.

**Comment by Gymnasiast** [comment]
I find a bottom note too intrusive. I'd prefer a tooltip.

**Comment by Nubbie** [comment]
Bottom note would be more visible for everyone, a tooltip would be more hidden as not many would hover over the language to see what it means

The Window is really small so wouldn't matter that much :o

**Comment by marijnvdwerf** [comment]
And by having it in the window, you could make the link clickable.

**Comment by Nubbie** [comment]
I would say:
```
STR_5830    :{SMALLFONT}{BLACK}Changes what language is used{Newline}Note: not all languages are complete
```
and in the bottom:
'Help us translate: http://Github.com/OpenRCT2/Localisation

**Comment by marijnvdwerf** [comment]
"Changes what language is used" is rather redundant for a dropdown with languages. If you _really_ need to use a tooltip, use one of those 'warning' ones with only 'Not all languages are complete' (though I reckon that should only be shown when an actually incomplete language has been selected.

As for the URL in the bottom, capitalisation is rather weird.

**Comment by Nubbie** [comment]
So if I get this straight:

The string should be:
```STR_5830    :{SMALLFONT}{BLACK}Changes what language is used```

When you select the language, a warning should pop up saying:
```Note: Not all languages are fully translated```

And in the bottom there should be a box called;
```Help us translate the games languages at:{NEWLINE}https://github.com/OpenRCT2/Localisation```
with Github OpenRCT2 being clickable

**Comment by Horsti12** [comment]
I don't think we longer need this enhancement, otherwise it would be already implemented in the past 2 years.

**Comment by Gymnasiast** [comment]
Probably too intrusive.
